scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python3
+import glob
+import os
+import argparse
+from git import Actor
+import shutil
+import git
+from git import Repo
+import json
+import wget
+import sys
+
+def parse_ver_json(commit_name):
+    # download versions.json and find commit name
+    try:
+        ver_url = 'https://testing.zephyrproject.org/daily_tests/versions.json'
+        ver_file = wget.download(ver_url, 'versions.json')
```

Comment:
maybe we can use urllib instead of wget so that we can add proxy
`
   ver_url = "https://testing.zephyrproject.org/daily_tests/versions.json"

    webURL = urllib.urlopen(ver_url)
    data = webURL.read()
    logging.info(json.loads(data)[-1])
    version = json.loads(data)[-1]
`